### ISSUE TITLE
Misalignment of footnote number and text in Preview

### ISSUE THREAD

**Issue by DutchPete** [akaalias/obsidian-footnotes#3]
I tested the plug-in (0.0.6) in a note with pre-existing inline footnotes. The "Alexis" (= normal 😄 ) footnote , number 2 in this case, between 2 pre-existing footnotes has its text on a line below the number - see screenshot below.

![image](https://user-images.githubusercontent.com/10794265/109799290-88c1c800-7c24-11eb-9426-5696d1964f65.png)

**Comment by akaalias** [comment]
Hey Klaas! This is in Preview, right? Can you share the .md and the PDF for me to check out?

**Comment by akaalias** [comment]
Also:

> The "Alexis" (= normal 😄 ) footnote

🤪😂

**Comment by DutchPete** [comment]
> Hey Klaas! This is in Preview, right? Can you share the .md and the PDF for me to check out?

Sent to you via a DM on the Obsidian forum.

**Comment by DutchPete** [comment]
Did you get the file?

**Comment by akaalias** [comment]
Yup!

**Comment by DutchPete** [comment]
The misalignment shown with a screenshot in the OP is NOT due to the plug-in. In the Obsidian default theme - no custom theme, no snippets - a footnote does not behave like that. So it is due to the custom theme I am using.

**Comment by DutchPete** [comment]
I reopen this issue because I made an interesting discovery.

I found that misalignment of the regular footnotes only occurs when there are pre-existing inline footnotes. I just applied regular footnotes to a note that had no inline footnotes, …… bingo, no misalignment.

So, if somehow existing inline footnotes could be converted in bulk to regular footnotes (something we touched on last Saturday), it is likely the misalignment would not be an issue anymore.

In any case, I don't think the misalignment has anything to do with the plug-in.
